packages/@aws-cdk/cx-api/lib/versioning.ts

```diff
@@ -16,7 +16,7 @@ import semver = require('semver');
  *   updated (as the current verison in package.json has already been released!)
  * - The request does not have versioning yet, only the response.
  */
-export const CLOUD_ASSEMBLY_VERSION = '0.36.0';
+export const CLOUD_ASSEMBLY_VERSION = '1.6.0';
```

Comment:
Isn't this a breaking change?

Reply:
It certainly means you will have to get a newer CLI version to use this functionality. Is that a breaking change? Does that mean we can never evolve the CX protocol ever again?

Reply:
I don't think this constitutes a breaking change as long as there's a good error (and there should be)